### PR TITLE
Update instrumentation.md

### PR DESCRIPTION
**Pull request by aengusrooneygrafana**
Update docs with client lib links as discussed with Annanay

**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/tempo/website/getting-started/instrumentation.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ weight: 100
 # OpenTelemetry
 
 Check out these resources for help instrumenting your favorite languages with OpenTelemetry instrumentation.  Most of these guides include complete
-end to end examples with Grafana, Loki, and Tempo.
+end to end examples with Grafana, Loki, and Tempo. 
+
+- [OpenTelemetry Language Specific Instrumentation] (https://opentelemetry.io/docs/)
+- [Jaeger Language Specific Instrumentation] (https://www.jaegertracing.io/docs/1.24/client-libraries/#supported-libraries)
```
Comment (joe-elliott) [inline docs/tempo/website/getting-started/instrumentation.md:14]: Use latest:

https://www.jaegertracing.io/docs/latest/client-libraries/

File: docs/tempo/website/getting-started/instrumentation.md [hunk 2]
```diff
@@ -8,7 +8,11 @@ weight: 100
 # OpenTelemetry
 
 Check out these resources for help instrumenting your favorite languages with OpenTelemetry instrumentation.  Most of these guides include complete
-end to end examples with Grafana, Loki, and Tempo.
+end to end examples with Grafana, Loki, and Tempo. 
+
+- [OpenTelemetry Language Specific Instrumentation] (https://opentelemetry.io/docs/)
```
Comment (joe-elliott) [inline docs/tempo/website/getting-started/instrumentation.md:13]: Perhaps create a new heading. Its weird these are under OpenTelemetry.

Comment (aengusrooneygrafana) [reply docs/tempo/website/getting-started/instrumentation.md:14]: Updated with a new heading now!

File: docs/tempo/website/getting-started/instrumentation.md (hunk 1 above)
Comment (aengusrooneygrafana) [reply docs/tempo/website/getting-started/instrumentation.md]: Updated with the link to 'latest' - good catch!

File: docs/tempo/website/getting-started/instrumentation.md (hunk 2 above)
Comment (dgzlopes) [reply docs/tempo/website/getting-started/instrumentation.md:14]:
```suggestion
- [OpenTelemetry Language Specific Instrumentation](https://opentelemetry.io/docs/)
```

File: docs/tempo/website/getting-started/instrumentation.md [hunk 3]
```diff
@@ -5,10 +5,19 @@ aliases:
...
-Check out these resources for help instrumenting your favorite languages with OpenTelemetry instrumentation.  Most of these guides include complete
-end to end examples with Grafana, Loki, and Tempo.
+Check out these resources for help instrumenting tracing with your favorite languages.  Most of these guides include complete end to end examples with Grafana, Loki, and Tempo. 
+
+## OpenTelemetry
+
+- [OpenTelemetry Language Specific Instrumentation] (https://opentelemetry.io/docs/)
+
+## Jaeger 
+- [Jaeger Language Specific Instrumentation] (https://www.jaegertracing.io/docs/latest/client-libraries/)
```
Comment (dgzlopes) [inline docs/tempo/website/getting-started/instrumentation.md:17]:
```suggestion
- [Jaeger Language Specific Instrumentation](https://www.jaegertracing.io/docs/latest/client-libraries/)
```

File: docs/tempo/website/getting-started/instrumentation.md [hunk 4]
```diff
@@ -5,10 +5,19 @@ aliases:
...
+
+## Zipkin 
+- [Zipkin Language Specific Instrumentation] (https://zipkin.io/pages/tracers_instrumentation.html)
```
Comment (dgzlopes) [inline docs/tempo/website/getting-started/instrumentation.md:20]:
```suggestion
- [Zipkin Language Specific Instrumentation](https://zipkin.io/pages/tracers_instrumentation.html)
```

**Review by dgzlopes — COMMENTED** [review]
Thanks for the PR!

Removed some spaces. Links weren't working for me on the Markdown render view.

**Review by mdisibio — APPROVED** [review]

**Pull request merged**